code/go/0chain.net/blobbercore/datastore/postgres.go

```diff
@@ -59,10 +63,63 @@ func (store *postgresStore) GetTransaction(ctx context.Context) *gorm.DB {
 	if conn != nil {
 		return conn.(*gorm.DB)
 	}
-	Logger.Error("No connection in the context.")
+	logging.Logger.Error("No connection in the context.")
 	return nil
 }
 
 func (store *postgresStore) GetDB() *gorm.DB {
 	return store.db
 }
+
+func (store *postgresStore) AutoMigrate() error {
+
+	err := store.db.AutoMigrate(&Migration{})
+	if err != nil {
+		logging.Logger.Error("[db]", zap.Error(err))
+	}
+
+	if len(releases) == 0 {
+		fmt.Print("	+ No releases 	[SKIP]\n")
+		return nil
+	}
+
+	for i := 0; i < len(releases); i++ {
+		v := releases[i]
+		fmt.Print("\r	+ ", v.Version, "	")
+		hasMigrated, err := store.HasMigrated(v)
+		if err != nil {
+			return err
+		}
+		if hasMigrated {
+			fmt.Print("	[SKIP]\n")
+		} else {
+			err = v.Migrate(store.db)
+			if err != nil {
+				logging.Logger.Error("[db]"+v.Version, zap.Error(err))
+				return err
+			} else {
+				logging.Logger.Info("[db]" + v.Version + " migrated")
+				fmt.Print("	[OK]\n")
+			}
+		}
```

Comment:
if hasMigrated {
			fmt.Print("	[SKIP]\n")
                        continue
		} 
		err = v.Migrate(store.db)
		if err != nil {
			logging.Logger.Error("[db]"+v.Version, zap.Error(err))
			return err
		} else {
			logging.Logger.Info("[db]" + v.Version + " migrated")
			fmt.Print("	[OK]\n")
		}

Reply:
do you mean add `continue` ?  BTW you can use ```suggestion ``` block. try it next time.

Reply:
improved loop code